addons/project/models/project.py

```diff
@@ -2182,6 +2181,9 @@ def _message_post_after_hook(self, message, msg_vals):
                     ('partner_id', '=', False),
                     ('email_from', '=', new_partner.email),
                     ('is_closed', '=', False)]).write({'partner_id': new_partner.id})
+        # use the sanitized body of the email from the message thread to populate the task's description
+        if not self.description and message.subtype_id.name == "Task Created" and self.partner_id == message.author_id:
```

Comment:
```suggestion
        if not self.description and message.subtype_id == self._creation_subtype() and self.partner_id == message.author_id:
```
since `mail.message.subtype().name` is a translated field we should check on XML ID.

If there was a default value, I guess these condition would also not work.

For that if there was the original body (but there is not) we could keep the original body and do:

```py
self.description == msg_vals.get('original_body')
```
but besides that I'm not sure how this could be done (unless we are sure that this is the message that created the thread, and so we can write without checking the content).

Reply:
Thanks for the review, I applied your suggestion for the usage of the XML-ID.

I've just confirmed with the PO that during creation there is no context in which the description already has a body, we just take the content of the email and it should be the description of the task/ticket. So I guess we don't have to worry about a default value, the email content should be the "default".